.github/workflows/ci.yml

```diff
@@ -510,6 +528,80 @@ jobs:
               body: 'Failed to pass the component examples UI Test. If this PR is for UI change and the error is snapshot mismatch, please add "ui change" label to the PR for updating the snapshot.'
             })
 
+  build_upload_base_bundle_stats:
```

Comment:
One hack I've seen is to store this kind of info as a gist. So everytime main CI runs it would update the gist containing our latest bundle sizes .. might be too hacky and also only works if the base ref is main but just throwing out an idea